src/parser/jobs/vpr/modules/SerpentsTail.tsx

```diff
@@ -74,9 +79,10 @@ export class SeprentsTail extends Analyser {
 	}
 
 	private onComplete() {
-		this.checklist.add(new Rule({
+		this.checklist.add(new TieredRule({
 			name: <Trans id = "VPR.serperntstail.wast.name"> Use your <DataLink action="SERPENTS_TAIL"/> follow-ups </Trans>,
```

Comment:
I'm going to hit you with the "while we're here" nit picks because these tags have a lot of inconsistency.
```suggestion
			name: <Trans id = "VPR.serpentstail.waste.name"> Use your <DataLink action="SERPENTS_TAIL"/> follow-ups </Trans>,
```